src/main/java/monitoringsystemturbo/presenter/ConfirmExportPresenter.java

```diff
@@ -62,13 +65,13 @@ public void setPrimaryStage(Stage primaryStage) {
 
     private void setCheckboxes() {
         boolean areAllCheckboxesSelected = areAllCheckboxesSelected();
-        for (CheckBox box : checkBoxes) {
+        for (JFXCheckBox box : checkBoxes) {
             box.setSelected(!areAllCheckboxesSelected);
         }
     }
 
     private boolean areAllCheckboxesSelected() {
-        for (CheckBox box : checkBoxes) {
+        for (JFXCheckBox box : checkBoxes) {
             if (!box.isSelected()) {
```

Comment:
you don't need to use if, just return what's inside if body

Reply:
I know that's not your change right now, but let's follow The Boy Scout Rule and improve code when seeing some smells